build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -29,6 +29,8 @@
     <PropertyGroup>
       <IsDesktopAvailable>False</IsDesktopAvailable>
       <IsDesktopAvailable Condition=" '$(OSName)' == 'win' ">True</IsDesktopAvailable>
+      <IsCross>False</IsCross>
+      <IsCross Condition=" !$(Rid.Contains('$(OSName)')) Or !$(Rid.Contains('$(Architecture)')) ">True</IsCross>
```

Comment:
I will use this feature in the next patch not to download debian packages and not to delete redist.dll and tool_msbuild.dll (https://github.com/dotnet/cli/blob/rel/1.0.0/build/Microsoft.DotNet.Cli.Compile.targets#L162#L163). Without these two dlls, our sdk didn't work normally.